packages/core/content-type-builder/admin/src/components/FormModal/forms/index.js

```diff
@@ -5,8 +5,84 @@ import { categoryForm, createCategorySchema } from '../category';
 import { contentTypeForm, createContentTypeSchema } from '../contentType';
 import { createComponentSchema, componentForm } from '../component';
 import { dynamiczoneForm } from '../dynamicZone';
+import { nameField } from '../attributes/nameField';
+import addItemsToFormSection from './utils/addItemsToFormSection';
+import getTrad from '../../../utils/getTrad';
+
+const getUsedAttributeNames = (attributes, schemaData) => {
```

Comment:
We can maybe move this to the utils folder since it already exists and this file is super long already